packages/actor-query-operation-path-one-or-more/lib/ActorQueryOperationPathOneOrMore.ts

```diff
@@ -19,17 +20,35 @@ export class ActorQueryOperationPathOneOrMore extends ActorAbstractPath {
   }
 
   public async runOperation(path: Algebra.Path, context: ActionContext): Promise<IActorQueryOperationOutputBindings> {
+    // Such connectivity matching does not introduce duplicates (it does not incorporate any count of the number
+    // of ways the connection can be made) even if the repeated path itself would otherwise result in duplicates.
+    // https://www.w3.org/TR/sparql11-query/#propertypaths
+    if (!context || !context.get('isPathArbitraryLengthDistinct')) {
+      context = context ?
+        context.set('isPathArbitraryLengthDistinct', true) :
+        Map.of('isPathArbitraryLengthDistinct', true);
```

Comment:
There should be an `ActionContext` function somewhere you can use for this instead.